staging/src/k8s.io/api/node/v1alpha1/types.go

```diff
@@ -114,3 +114,69 @@ type RuntimeClassList struct {
 	// Items is a list of schema objects.
 	Items []RuntimeClass `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
+// +k8s:conversion-gen:explicit-from=net/url.Values
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeLogViewerOptions is the query options for a Node's logs REST call.
+type NodeLogViewerOptions struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// SinceTime is an RFC3339 timestamp from which to show logs. /If this value precedes the time a log file or service
```

Comment:
What does `/If` mean?

Reply:
The "If this value precedes the time a log file or service was created.." clause seems like it is either obvious (in which case why is it documented?) or so subtle in its meaning that I am not understanding it.

Of course it "only shows logs since the service was created" - what else would it do?

Reply:
Yes, this is just stating the obvious. I'll remove the sentence.